### PR TITLE
Line clamp on carousel heading

### PR DESCRIPTION
**Pull request by curtisadams28**
## Description of work
- Adds a mixin that adds the line-clamp property with the number of lines as an argument.
- Line clamped the carousel heading to 3 lines.

## Testing
- Create a heading that is long enough that it should wrap to 4 lines.

## Checklist
- [x] CI tests are passing Github actions (inc. linting)
- [x] Key areas of the feature outlined for context and testing
- [x] If there are designs for this work are they noted here and in the ADO card 
- [x] Design review
- [ ] Have checked this at multiple screen resolutions and range of browsers
- [x] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [x] Updated changelog (if applicable)
- [x] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

**Comment by PeculiarGoat** [comment]
Is it worth adding a mixin that only gets used once?

**Review by smitsanghvi21 — COMMENTED** [review]
Upon having a brief look couldn't really see a visual difference on how the heading is rendering after/before. Not sure If I am missing something.

**Comment by PeculiarGoat** [comment]
Is there an actual plan to use it?  If not, then we can probably live without the additional complexity for now.

**Comment by stajs** [comment]
Will the film detail page need it?

**Comment by curtisadams28** [comment]
@stajs Maybe. A long title could push the rest of the content below the fold. I've removed the mixin for now and it can be added back later if we need it.

**Comment by PeculiarGoat** [comment]
Have you checked those properties on caniuse?

**Comment by curtisadams28** [comment]
Yes. I've also tested in Chrome and Firefox. Still need to test on IOS.

**Comment by stajs** [comment]
[Support is good for `-webkit-line-clamp`](https://caniuse.com/mdn-css_properties_-webkit-line-clamp), suprisingly so even for non-Webkit browsers.

If you look at [the non-prefix "official" standard `line-clamp`](https://caniuse.com/css-line-clamp), they push you to the prefix:

image

**Comment by brynyShift72** [comment]
Apart from iOS is that everything that needs to be checked?
Are you wanting to merge this with 1.0.1 or after?

File: site/styles/_carousel.scss [hunk 1]
```diff
@@ -131,6 +131,13 @@ s72-carousel {
     max-width: 930px;
     padding-bottom: 0;
 
+    /* stylelint-disable */
+    display: -webkit-box;
+    overflow: hidden;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
```
Comment (PeculiarGoat) [inline site/styles/_carousel.scss:138]: Do we care that `box-orient` is deprecated?

**Review by PeculiarGoat — CHANGES_REQUESTED** [review]
I merged in develop + ran the linter.

File: site/styles/_carousel.scss (hunk 1 above)
Comment (stajs) [reply site/styles/_carousel.scss:138]: I care, but MDN is the one reporting it deprecated, and yet [use it in their own example](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp). 🤷

**Comment by stajs** [comment]
@curtisadams28 just checking: @PeculiarGoat requested changes, not sure what they were but you had confirmed it was okay?

**Comment by stajs** [comment]
@curtisadams28 just checking: @PeculiarGoat requested changes, not sure what they were but you had confirmed it was okay?

**Pull request merged**